app/views/schools/advice/boiler_control/analysis.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= render 'schools/advice/advice_page', school: @school, advice_pages: @advice_pages, advice_page: @advice_page, tab: @tab do %>
+  <h2><%= t("advice_pages.#{@advice_page.key}.analysis_title") %></h2>
+<% end %>
```

Comment:
I was reading here: https://guides.rubyonrails.org/v6.1/layouts_and_rendering.html#template-inheritance that "Similar to the Layout Inheritance logic, if a template or partial is not found in the conventional path, the controller will look for a template or partial to render in its inheritance chain". So we could potentially have an analysis.html.erb in /advice with the above in, that rendered a partial called _analysis.html.erb for the page content, which would be found in the page (boiler_control in this case) sub-directory. Would need to try it though!